packages/x-components/src/main.ts

```diff
@@ -6,10 +6,21 @@ import { baseInstallXOptions, baseSnippetConfig } from './views/base-config';
 import { XInstaller } from './x-installer/x-installer/x-installer';
 import { FilterEntityFactory } from './x-modules/facets/entities/filter-entity.factory';
 import { SingleSelectModifier } from './x-modules/facets/entities/single-select.modifier';
+import { StickyModifier } from './x-modules/facets/entities/sticky.modifier';
 import './tailwind/index.css';
 
 Vue.config.productionTip = false;
 FilterEntityFactory.instance.registerModifierByFacetId('age_facet', SingleSelectModifier);
+FilterEntityFactory.instance.registerModifierByFacetId(
+  'brand_facet',
+  SingleSelectModifier,
+  StickyModifier
```

Comment:
I find it awkward to send a list of modifiers like this instead of with a list, also considering one could override the next. Should we maintain this method signature or change to a list?